src/Startup.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace GroupDocs.Metadata.WebForms
+{
+    public partial class Startup {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-WebForms/pullRequest?prid=5102156)